.travis-deploy-test.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/sh
+
+# Immediately cancel if there is no deployment key
+if [ -z "${STUDIO_TEST_DEPLOY_KEY}" ]; then
+  echo No deployment key. Canceling deployment.
+  exit 0
+fi
+
+set -eu
+
+# Prepare GitHub SSH key
```

Comment:
This comment seems misplaced. I would probably move the `if else` below further down.

Reply:
fixed